primer/brand
# InlineLinks inherit text size even with custom css

## Summary

Inline Links will now inherit their text size, even when custom CSS is used. Fixes #446 

## List of notable changes:


- **updated** Inline Links will now inherit their text size, even when custom CSS is used.


## What should reviewers focus on?



## Steps to test:

1. Go to `components-inlinelink--size-inheritence-with-custom-css` in Storybook
2. Verify that the font-size is being inheriting correctly.

## Supporting resources (related issues, external links, etc):

- https://developer.mozilla.org/en-US/docs/Web/CSS/inherit


## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![CleanShot 2023-10-16 at 13 07 55@2x](https://github.com/primer/brand/assets/12260694/3cf27600-46cc-449f-a291-a85c4d86f9ad)



 </td>
<td valign="top">

![CleanShot 2023-10-16 at 13 07 32@2x](https://github.com/primer/brand/assets/12260694/9d0a29bd-3fcf-4c85-a915-72a01fb7e936)



</td>
</tr>
</table>
